docs/swiper.md

```diff
@@ -0,0 +1,500 @@
+---
+id: swiper
+title: Swiper
+---
+
+### Usage
+
+```js
+import { Swiper } from 'react-native-elements'
+
+<Swiper>
+  <View style={{ flex: 1 }}>{/* any content */}</View>
+  <View style={{ flex: 1 }}>{/* any content */}</View>
+  {/* ... */}
+</Swiper>
+
+<Swiper
+  vertical {/* slide up / down instead left / right */}
+  from={1} {/* initial slide is second */}
+  loop {/* back to first slide after last */}
+  timeout={2} {/* autoplay duration (2 secs) */}
+  springConfig={{ speed: 11 }} {/* RN Animated.spring config */}
+  minDistanceForAction={0.15} {/* Swipe less that 15% keep active slide */}
+  positionFixed {/* Fix mobile safari vertical bounces */}
```

Comment:
[Just for example] - Leave prop comments outside of the example, there's already a reference section. Also, remove any props that aren't necessary for it to work really.

So:
```
  timeout={2} {/* autoplay duration (2 secs) */}
  springConfig={{ speed: 11 }} {/* RN Animated.spring config */}
  minDistanceForAction={0.15} {/* Swipe less that 15% keep active slide */}
  positionFixed
```